openfast_toolbox/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -1890,11 +1897,17 @@ def TS_high_get_time_series(self):
                         uvel = np.roll(bts['u'][0, :, jTurb, kTurb], start_time_step)
                         vvel = np.roll(bts['u'][1, :, jTurb, kTurb], start_time_step)
                         wvel = np.roll(bts['u'][2, :, jTurb, kTurb], start_time_step)
+
+                        # Map it to high-res time and dt (both) 
+                        time_hr = np.arange(0, self.tmax + self.dt_high, self.dt_high)
```

Comment:
Can we have this array be built using the `time` array? If the overall `tmax` is already aligned between high- and low-res, can we have something like below?
`time_hr = np.arange(time[0], time[-1], self.dt_high)`

Reply:
nope, we cannot. I had it this way, and I don't understand why, but turbsim plays some games and the boxes read into python have a `time` that, at times, is longer than the `analysis time` set by the user... try it and you will see